core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -139,6 +179,22 @@ default void onNodeSuccess(
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node) {}
 
+  /**
+   * @deprecated This method only exists for backward compatibility. Override {@link
+   *     #onNodeSuccess(Request, long, DriverExecutionProfile, Node, String, ExecutionInfo)}
+   *     instead.
+   */
+  default void onNodeSuccess(
```

Comment:
You're missing the `@Deprecated` annotation mentioned in the comment on this one